packages/amplify-codegen/package.json

```diff
@@ -20,6 +20,7 @@
     "test-watch": "jest --watch"
   },
   "dependencies": {
+    "@aws-amplify/appsync-modelgen-plugin": "^1.20.12-dev.136",
```

Comment:
The version number should be "@latest"

Reply:
changes to latest

Reply:
I was reading about this and it is considered a good practice to not always depend on latest version of dependency package but lock it down to the latest major version. If we want to do that, we have to change it to `"@aws-amplify/appsync-modelgen-plugin": "^1.x"`. @yuth what do you think?